### PR TITLE
add r-latticeextra  r-mapproj  r-markdown  r-memoise  r-minqa  r-modelmetrics  r-munsell

### PR DESCRIPTION
**Pull request by bgruening**
r-latticeextra
 r-mapproj
 r-markdown
 r-memoise
 r-minqa
 r-modelmetrics
 r-munsell

xref: https://github.com/conda-forge/staged-recipes/issues/2009

### CODE REVIEW

**Comment by bgruening** [comment]
@isuruf green! If you need something to test.

**Comment by isuruf** [comment]
Thanks, but somebody else will have to review this, because I'm not familiar with the r packages.

File: recipes/r-latticeextra/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```
Comment (isuruf) [inline recipes/r-latticeextra/build.sh]: What does this do? Is this needed only for unix?

File: recipes/r-latticeextra/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+source:
+  fn: latticeExtra_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/latticeExtra_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/latticeExtra/latticeExtra_{{ version }}.tar.gz
+  sha256: 780695323dfadac108fb27000011c734e2927b1e0f069f247d65d27994c67ec2
+
+build:
+  number: 0
+  # r-maps is missing in win64 as well
```
Comment (isuruf) [inline recipes/r-latticeextra/meta.yaml]: If `r-maps` is a dependency of this package, then add it to build and run requirements

File: recipes/r-latticeextra/build.sh (hunk 1 above)
Comment (bgruening) [inline recipes/r-latticeextra/build.sh]: This is from the skeleton generator.

**Comment by bgruening** [comment]
AppVeyor seems also to be finished and green. So we need to retrigger a build?

**Comment by bgruening** [comment]
ping @jakirkham any idea?

**Comment by isuruf** [comment]
@bgruening, if somebody reviews this PR, this is okay to merge.

**Comment by bgruening** [comment]
@johanneskoester @jakirkham ready for merging.

**Pull request merged**